### ISSUE TITLE
Peer review for hw06-garyzhubc

### ISSUE THREAD

**Issue by ziqiangt** [STAT545-UBC-hw-2018-19/hw06-garyzhubc#2]
|    **Topic**    | **Excellent** |**Satisfactory** |**Needs Work** |
|----------------|---------------|------------------|------------------|
| Coding style  |   :thumbsup:    |                        |                         |
| Coding strategy      |    |         :thumbsup:                 |                           |
| Presentation: graphs | |                          |            😟               |
| Presentation: tables |  |                         |            😟              |
| Achievement, activity |    |                        |          😟                 |
| Ease of access |   |        :thumbsup:             |                       |

Remarks:
garyzhubc seems not finish the required task. Here are some comments that might interest you. 
1. You could add a readme file and this would be really kind for the new arriver to go through what you did
2. It seems that there is not enough code chunk comments
3. You could try this code to make your md file more readable: 
output:
  html_document:
    keep_md: true
    toc: true

In conclusion, it is a good answer.
